test/e2e/basic.browser.test.js

```diff
@@ -28,10 +28,12 @@ describe('basic browser', () => {
   })
 
   test('/noloading', async () => {
-    const { hook } = await page.nuxt.navigate('/noloading')
-    const loading = await page.nuxt.loadingData()
+    const { hook } = await page.nuxt.navigate('/noloading', false)
+    await Utils.waitFor(nuxt.options.loading.throttle + 100)
```

Comment:
Could we maybe move this right into `page.nuxt.navigate` @pimlie ?

Reply:
Yes, or we could add a call to `page.waitForFunction` instead. But as mentioned I actually think the e2e tests need some more refactoring anyway as there doesnt seem to be enough protection against unexpected exceptions at the moment. Maybe we can change this with that refactoring?